src/main/java/com/hillel/items_exchange/model/Category.java

```diff
@@ -0,0 +1,25 @@
+package com.hillel.items_exchange.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@EqualsAndHashCode(exclude = "subcategories")
+@Entity
+@Data
```

Comment:
@Data needs to be removed and getters, setters annotation have to be added next time.